runtimes/pythonrt/runner/syscalls.py

```diff
@@ -100,10 +100,17 @@ def ak_subscribe(self, args, kw):
         return resp.signal_id
 
     def ak_next_event(self, args, kw):
-        (id,) = extract_args(["subscription_id"], args, kw)
+        log.info("ak_next_event: %r %r", args, kw)
+        (
+            id,
+            timeout,
+        ) = extract_args(["subscription_id", "timeout?"], args, kw)
         if not id:
             raise ValueError("empty subscription_id")
+
         req = pb.NextEventRequest(runner_id=self.runner_id, signal_ids=[id])
+        if timeout:
+            req.timeout_ms = int(timeout * 1000)
```

Comment:
In `py-sdk/events.py`, the type of the timeout is `timedelta`, not `int`, which actually makes sense.

BTW, unrelated to your PR, do you think we should make `timeout` a kwarg-only parameter? (i.e. `def next_event(id, *, timeout)`)